.github/workflows/tests.yml

```diff
@@ -0,0 +1,109 @@
+name: Tests, Behat, Coding Standards
+on: push
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.2', '7.3']
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: ${{ matrix.php-versions }}
+        extensions: mbstring, intl
+        ini-values: post_max_size=256M, short_open_tag=On
+        tools: phpunit, composer
+    - name: Run tests
+      run: |
+        composer install
+        ./vendor/bin/phpunit -c ./phpunit.xml.dist
+  test-5-4:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '5.6'
```

Comment:
Ist das nun ein Test für PHP 5.4 oder 5.6? Würde auch ein Test mit PHP 5.4 funktionieren?
Is this a test for PHP 5.4 or 5.6? Would it also run with 5.4? If not, I think we can just rename it (and then also rename the `composer-php-5-4.json`?).

Reply:
Let’s change it to 5.6, and also change the `php` version number in the `composer-php-5-4.json`

Reply:
After switching it to 5.6 the test failed :D So I switched it back to 5.4, but also the PHP version of the runner. Now it works again.